package.json

```diff
@@ -24,6 +24,7 @@
     "openlayers": "openlayers/ol3#master",
     "proj4": "2.3.3",
     "phantomjs": "~1.9.7-5",
+    "typeahead.js": "~0.10.5",
```

Comment:
No it's not for the moment. It is just a [devDependency](https://github.com/camptocamp/ngeo/blob/master/package.json#L38). It may be a mistake in ngeo though…